upload/upload_test.go

```diff
@@ -438,3 +423,74 @@ func (suite *TestSuite) TestRecursiveToDifferentTarget() {
 
 	log.SetOutput(os.Stdout)
 }
+
+func (suite *TestSuite) TestUploadInvalidCharacters() {
+	// Filenames with :\?* can not be created on windows
+	if runtime.GOOS == "windows" {
+		return
```

Comment:
```suggestion
		suite.T().Skip()
```

You should be able to use something like the `skip` in order to show that the test where skipped rather than give the impression that it passed.

Reply:
Good point! Done in 260f1d8:

![2023-11-28-160019_839x36_scrot](https://github.com/NBISweden/sda-cli/assets/1029190/607c4afa-e637-4a1c-86c5-aa86c615fded)